tests/forte/data/extractor/seqtagging_extractor_test.py

```diff
@@ -0,0 +1,70 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#       http://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import unittest
+
+from ft.onto.base_ontology import Sentence, Token, EntityMention
+from forte.pipeline import Pipeline
+from forte.data.readers.conll03_reader import CoNLL03Reader
+from forte.data.data_pack import DataPack
+from forte.data.extractor.seqtagging_extractor import BioSeqTaggingExtractor
+
+
+class SeqTaggingExtractorTest(unittest.TestCase):
+
+    def setUp(self):
+        # Define and config the Pipeline
+        self.dataset_path = "data_samples/conll03"
+
+    def test_BioSeqTaggingExtractor(self):
+        pipeline = Pipeline[DataPack]()
+        reader = CoNLL03Reader()
+        pipeline.set_reader(reader)
+        pipeline.initialize()
+
+        config = {
+            "entry_type": EntityMention,
+            "need_pad": True,
+            "attribute": "ner_type",
+            "based_on": Token,
+        }
+
+        expected = [(None, 'O'), ('ORG', 'B'), ('ORG', 'I'),
+                    (None, 'O'), (None, 'O'), (None, 'O'),
+                    (None, 'O'), (None, 'O'), (None, 'O'),
+                    ('MISC', 'B'), (None, 'O'), (None, 'O'),
+                    (None, 'O'), (None, 'O'), (None, 'O'),
+                    ('MISC', 'B'), (None, 'O'), (None, 'O'),
+                    (None, 'O'), (None, 'O'), (None, 'O'),
+                    (None, 'O'), (None, 'O'), (None, 'O'),
+                    (None, 'O'), (None, 'O'), (None, 'O'),
+                    (None, 'O'), (None, 'O'), (None, 'O')]
+
+        extractor = BioSeqTaggingExtractor(config)
+
+        for pack in pipeline.process_dataset(self.dataset_path):
+            for instance in pack.get(Sentence):
+                extractor.update_vocab(pack, instance)
+
+        for pack in pipeline.process_dataset(self.dataset_path):
+            for instance in pack.get(Sentence):
+                feature = extractor.extract(pack, instance)
+                recovered = [extractor.id2element(idx) for idx in feature._data]
+                print(recovered)
+                self.assertListEqual(expected, recovered)
+                extractor.add_to_pack(pack, instance, feature._data)
+            pack.add_all_remaining_entries()
```

Comment:
Look, this actually indicates why we shouldn't call the extractor as the library users. Maybe we should make the documentation clear.

Reply:
Yes. Right.
Here is for the test code, so I call the `add_to_pack` function. Where should I add more documentation?

Reply:
I think mainly at the high level extractor classes, at the very top, we can specify something like "These classes should not be called directly by the users"

Reply:
I add "this class is used internally" in each extractor file.